src/GeositeFramework/plugins/layer_selector_v2/plugin.json

```diff
@@ -0,0 +1,8 @@
+{
+    "css": [
+        "plugins/layer_selector_v2/main.css"
+    ],
+    "use": {
+        "underscore": { "attach": "_" }
```

Comment:
Underscore is actually AMD compatible now, I'll try and see if I can figure out how to get it without a shim at all.